gtfs-validator/main/src/main/java/org/mobilitydata/gtfsvalidator/notice/MissingTripEdgeStopTimeNotice.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 Google LLC, MobilityData IO
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.notice;
+
+import com.google.common.collect.ImmutableMap;
+
+// Trip edge = first or last stop 
+public class MissingTripEdgeStopTimeNotice extends Notice {
+  public MissingTripEdgeStopTimeNotice(
+      String fieldName,
+      String tripId, 
+      int csvRowNumber,
+      int stopSequence) {
+    super(
+        ImmutableMap.of(
+            "fieldName", fieldName,
```

Comment:
Maybe name it directly as "arrival_or_departure_time" to be clearer?

Reply:
Yep good idea, done now